proarc-webapp/src/main/java/cz/cas/lib/proarc/webapp/client/widget/workflow/WorkflowJobFormView.java

```diff
@@ -413,7 +413,7 @@ public void performAction(ActionEvent event) {
     }
 
     private Widget createMaterialList() {
-        materialView = new WorkflowMaterialView(i18n, true);
+        materialView = new WorkflowMaterialView(i18n, true, this.getClass().getSimpleName());
```

Comment:
Vrátí to "WorkflowJobFormView". Metody getClass a getSimpleName GWT od verze 2.6.0 RC1  emuluje (http://www.gwtproject.org/doc/latest/RefJreEmulation.html).

Líbilo se mi, že nemusím zavádět další řetězec.